demo/shaders/compile_shaders.bat

```diff
@@ -3,10 +3,10 @@ setlocal
 cd /D "%~dp0"
 
 cargo run --package rafx-shader-processor -- ^
---glsl-path glsl/*.vert glsl/*.frag glsl/*.comp ^
```

Comment:
GL ES 3.0 doesn't support compute shaders. We could take an approach of having all the code use 3.0, except anything having to do with compute shaders, which will use 3.1. This would be a runtime check and we'd compile compute GLSL as 3.1. It would be a runtime error to use compute when it's not available. We should add a "compute_available" flag to the device info. Maybe we should add a few more flags for other things (like shader addressing mode, metal only recently added clamp to border color)